src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/SetAzureCognitiveServicesAccount.cs

```diff
@@ -139,6 +139,12 @@ public string KeyVaultUri
         [AllowEmptyCollection]
         public PSNetworkRuleSet NetworkRuleSet { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The network access type for Cognitive Services Account.")]
+        [ValidateNotNull]
```

Comment:
please add
[ValidateSet("Enabled", "Disabled", IgnoreCase = true)]

and also add possible values to help message